js/localization.d.ts

```diff
@@ -117,4 +117,11 @@ type ExternalFormat = any;
  * @namespace DevExpress.ui
  * @public
  */
-export type Format = FormatObject | PredefinedFormat | string | ((value: number | Date) => string) | ExternalFormat;
+export type Format =
+  FormatObject |
+  PredefinedFormat |
+  string |
+  ((value: number | Date) => string) |
+  ((value: Date) => string) |
+  ((value: number) => string) |
```

Comment:
I suggest to get rid of 1 or 2 and 3

Reply:
Removing these signatures breaks renovation compilation: I cannot pass a delegate with the `(value: Date) => string` signature as an argument to a function that expects the following one: `(value: number | Date) => string`